articles/openshift/oshift-faq.md

```diff
@@ -65,7 +67,7 @@ We monitor, maintain and support our controlled UKCloud for OpenShift infrastruc
 
 ### Can I use UKCloud for OpenShift in the UKCloud Elevated (previously IL3) domain?
 
-UKCloud for OpenShift services are available in both our Assured (previously IL2) and Elevated (previously IL3) security domains.
+UKCloud for OpenShift services are available in both our Assured (previously IL2) and upon request in our Elevated (previously IL3) security domains.
```

Comment:
I think we can get rid of the previously IL2/IL3 bit now

Reply:
Updated accordingly